docs/userguide/dependency_management.txt

```diff
@@ -122,69 +195,104 @@ temporary folder and run ``setup.py bdist_egg``.
 
 The ``dependency_links`` option takes the form of a list of URL strings.  For
 example, this will cause a search of the specified page for eggs or source
-distributions, if the package's dependencies aren't already installed::
+distributions, if the package's dependencies aren't already installed:
+
+.. code-block:: ini
+
+    [options]
+    #...
+    dependency_links = http://peak.telecommunity.com/snapshots/
+
+.. code-block:: python
 
     setup(
-        ...
+        #...
         dependency_links=[
             "http://peak.telecommunity.com/snapshots/"
         ],
     )
 
 
-.. _Declaring Extras:
+Optional dependencies
+=====================
+Setuptools allows you to declare dependencies that only get installed under
```

Comment:
Somewhere early, I'd mention that optional dependencies are also great for declaring dependencies for ancillary functions such as 'tests' and 'docs' and also make a note that `tests_require` is deprecated (similar to the note about `setup_requires` above.